@navikt/core/react/src/overlays/floating-menu/Menu.tsx

```diff
@@ -1067,11 +920,6 @@ const MenuSubContent = forwardRef<
         // The menu might close because of focusing another menu item in the parent menu. We
         // don't want it to refocus the trigger in that case so we handle trigger focus ourselves.
         onCloseAutoFocus={(event) => event.preventDefault()}
```

Comment:
Is this still needed? If so, then maybe the comment needs to be updated.

Reply:
Still needed, but updated comment 👍